### DISCUSSION TITLE
Maximum path length limit

### DISCUSSION THREAD

**Discussion by CBle87** [winpython/winpython#1022] [Q&A] [answered]
Hi,
When opening a file with _open(filename)_ I noticed a limitation of _len(filename)<=260_.
Whenever exceeding this length the interpreter pops up an error which dates back to the older days of windows.
Is there any chance to disable this limitation in winpython?
Cheers!
Christian

**Comment by stonebig** [comment]
Isn't it a limitation you can disable via windows  regedit ?

**Reply by CBle87** [reply]
@stonebig You are absolutely right, thank you! :)

**Comment by CBle87** [comment, marked as answer]
Solution:

1) run _regedit_ as admin

2) navigate to _HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem_

3) change _LongPathsEnabled_ from 0 to 1
